src/test/java/com/amazon/opendistroforelasticsearch/search/asynchronous/service/AsynchronousSearchServiceTests.java

```diff
@@ -87,7 +89,7 @@
 
     private ClusterSettings clusterSettings;
     private ExecutorBuilder<?> executorBuilder;
-
+    static boolean blockPersistence;
```

Comment:
I don't think so this should be static

Reply:
It's used in a static class - `mockClient`.